MODULE.bazel

```
@@ -2,7 +2,18 @@
 # Exceptions. See /LICENSE for license information.
 # SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
 
-"""Bazel modules."""
+"""Bazel modules.
+
+If `MODULE.bazel.lock` changes locally, it means the host platform is not part
+of the GitHub supported set. Running `bazel mod deps` provides a canonical
+update to `MODULE.bazel.lock`; create a PR with those changes in order to
+include the host platform.
+
+Normally, test actions are used to require updates on platforms that GitHub will
+run tests on. Right now that's linux amd64 and osx x86_64. Because other
+platforms are not covered by CI, they may fall out of sync on dependency
+changes.
```

I had some trouble understanding this, what do you think about an alternate phrasing?

```suggestion
Platforms tested with GitHub actions are kept up-to-date. Other platforms may fall out of sync on dependency
changes, and should be updated with a PR the same way a platform is added.
```